Doc/library/unittest.mock.rst

```diff
@@ -737,16 +737,16 @@ the *new_callable* argument to :func:`patch`.
 
     .. attribute:: __class__
 
-        Normally the :attr:`__class__` attribute of an object will return its type.
-        For a mock object with a :attr:`spec`, ``__class__`` returns the spec class
+        Normally the :attr:`!__class__` attribute of an object will return its type.
+        For a mock object with a :attr:`spec`, :attr:`!__class__` returns the spec class
```

Comment:
The `spec` attribute seems to be missing or is it an inherited one?

Reply:
I think it (and `spec_set`) are just missing documentation under https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock

For example, here's `side_effect`:

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.side_effect

Would you like to suggest some text?

Reply:
I never used it so I cannot really suggest a good text. I'll leave that task to unittest experts! but we can do it in another PR otherwise.